qiskit/providers/aer/backends/wrappers/bindings.cc

```diff
@@ -20,21 +20,33 @@ DISABLE_WARNING_POP
 #include "controllers/unitary_controller.hpp"
 #include "controllers/controller_execute.hpp"
 
-PYBIND11_MODULE(controller_wrappers, m) {
-
-    m.def("qasm_controller_execute_json", &AER::controller_execute_json<AER::Simulator::QasmController>, "instance of controller_execute for QasmController");
```

Comment:
I guess we don't use the `controller_execute_json` functions. Shouldn't we remove then their C++ counterparts: `AER::controller_execute_json<AER::Simulator::QasmController>`, etc?

Reply:
Sure - done in 7bb6843